MicrosoftFluentUI.podspec

```diff
@@ -12,8 +12,10 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = "11.0"
   s.ios.source_files   = "ios/FluentUI/**/*.{swift,h}"
   s.ios.resource_bundle = { 'FluentUIResources-ios' => "ios/FluentUI/**/*.{storyboard,xib,xcassets,strings,stringsdict}" }
```

Comment:
is this correct? this has to be on the same s.ios.resource_bundle declaration. otherwise I don't think clients can load the resources correctly.